src/components/Home.js

```diff
@@ -1,26 +1,37 @@
 import React from 'react';
 import Beer from './Beer';
 
-class Home extends React.Component {
```

Comment:
Any component without state or lifecycle methods can be made into a stateless functional component.